### DISCUSSION TITLE
Pimcore 11 : admin issue with assigning multiple roles to the user

### DISCUSSION THREAD

**Discussion by harshida-patel** [pimcore/pimcore#17631] [General]
Pimcore allows assigning multiple roles to a user. If one role has access to workflow_details and another role lacks access, the user will not have access to workflow_details when both roles are assigned.

**Comment by kingjia90** [comment]
this line should ensure that whenever one roles grants the permission, it would be positive, no matter the conflicting permissions that negate the access
https://github.com/pimcore/pimcore/blob/ef202e192e83191dcbb33ebd676f78004b37cfa2/models/User.php#L277-L279

Could you please debug from there to see if it's not working as intended? Maybe clearing cache?
